src/ProjectTemplates/Quantum.Quantinuum.App1/.template.config/template.json.v.template

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Looks like inside of the dir "src/ProjectTemplates" you renamed the dir 
"Quantum.**Honeywell**.App1" to 
"Quantum.**Quantinuum**.App1" _manually_ or using an IDE (rather than using `git mv`). That's why the file ".template.config/template.json.v.template" inside of the renamed dir is shown in the diff as fully removed and then new file added. As a result the history of the file changes (and line changes in it) gets lost.

If situation above is the case then consider using the `git mv` to rename the dir. Then commit (without making any other edits). Then make changes to the file. Then commit. To summarize, keep renaming in a commit separate from file changes.

Reply:
Thanks, Great point, I'll revert the changes and do as suggest.